.bazelci/integration.yml

```diff
@@ -1,10 +1,23 @@
+lts: &lts
+  bazel: latest
+
+rolling: &rolling
+  bazel: rolling
+
+
 common: &common
-  name: distribution
+  platform: ubuntu1804
   working_directory: /workdir/pkg
   build_targets:
   - "distro:*"
 
 tasks:
-  ubuntu1804:
+  rolling_ubuntu:
+    name: rolling_distro
+    <<: *rolling
+    <<: *common
+  lts_ubuntu:
```

Comment:
Amusingly, Ubuntu 18.04 could also be considered "LTS ubuntu", so perhaps the name could be clarified a bit?

It's clear to me what's going on (and buildkite looks fine), so I don't feel too strongly about it.